### DISCUSSION TITLE
RDR2

### DISCUSSION THREAD

**Discussion by Kotetsu86** [NarutoUA/gta5_fsr#47] [Ideas]
How hard would it be to make this mod work for RDR2 its the same engine right?

**Comment by DarkwaterV3** [comment]
I too would like to know. Thanks in advance for your thoughts, NarutoUA.

**Comment by NarutoUA** [comment]
Its not the **same** engine. It is next gen of RAGE which uses DirectX12/Vulkan instead of DX10/11 (GTA5) or DX9(GTA4). I've already started RDR2 mod but I dont have motivation to continue because I have no DX12/Vulkan experience, I've never been interested in this game and some other reasons I am not ready to say yet.

**Reply by DarkwaterV3** [reply]
Sorry to hear that. Thanks for the reply and good luck if you're ever gonna give it a go!
